gooey/gui/components/widgets/core/chooser.py

```diff
@@ -64,40 +64,57 @@ def getValue(self):
 class FileChooser(Chooser):
     """ Retrieve an existing file from the system """
     def getDialog(self):
-        return wx.FileDialog(self, _('open_file'), style=wx.FD_OPEN | wx.FD_FILE_MUST_EXIST)
+        options = self.Parent._options
+        return wx.FileDialog(self, message=options.get('message', _('open_file')),
+                             style=wx.FD_OPEN | wx.FD_FILE_MUST_EXIST,
+                             defaultFile=options.get('defaultFile', _("enter_filename")),

```

Comment:
You'll need to add these new options to the Gooey Options docs here: https://github.com/chriskiehl/Gooey/blob/1.0.3-release/docs/Gooey-Options.md#individual-widget-options

Reply:
I've added documentation.
Please review it is clear.